jetty-core/jetty-tests/jetty-test-client-transports/src/test/java/org/eclipse/jetty/test/client/transport/HttpClientStreamTest.java

```diff
@@ -1193,6 +1202,109 @@ public boolean handle(Request request, org.eclipse.jetty.server.Response respons
         assertTrue(clientLatch.await(timeoutInSeconds, TimeUnit.SECONDS));
     }
 
+    @ParameterizedTest
+    @MethodSource("transports")
+    @Tag("DisableLeakTracking:server:UNIX_DOMAIN")
```

Comment:
Please restore the tag disabling leak tracking for server HTTP and HTTPS as those are flaky:

```
@Tag("DisableLeakTracking:server:HTTP")
@Tag("DisableLeakTracking:server:HTTPS")
```

Reply:
I did this.

@lachlan-roberts but there are still other tests failing.